decidim-elections/app/models/decidim/votings/polling_station_closure.rb

```diff
@@ -4,7 +4,7 @@ module Decidim
   module Votings
     # The data store for an Election Closure.
     class PollingStationClosure < ApplicationRecord
-      enum phase: [:envelopes, :results, :attachment, :confirmed, :freeze], _suffix: true
+      enum phase: [:envelopes, :results, :attachment, :sign, :freezed], _suffix: true
```

Comment:
💬 I find these names a bit confusing (and I think it should be frozen instead of freezed, but I'm not sure). I'd use something like: `count` (or `totals`, `ballots` or `ballots_count`), `results`, `certificate`, `signature` and `complete`. Please, feel free to use any of these names only if you think they are clearer than the current ones.

Reply:
Thanks, I like your suggestions I'll update it to:
`count`, `results`, `certificate`, `signature` and `complete`.